public/js/index.js

```diff
@@ -187,7 +190,7 @@ function loadTagSuggestions() {
 
 			// Perform a search when a tag is selected
 			Awesomplete.$('#srch').addEventListener("awesomplete-selectcomplete", function() {
-				arcTable.search($('#srch').val()).draw();
+				favTagSearch();
```

Comment:
Since this is used for all forms of search, calling the function `favTagSearch()` is now misleading; The function should be renamed to something like `performSearch()` and the comment on line 43/44 updated accordingly.

Reply:
Updated